src/fidasim.f90

```diff
@@ -6117,8 +6217,28 @@ subroutine gyro_step(vi, fields, r_gyro)
         else
             rg_uvw = matmul(beam_grid%basis,r_gyro)
         endif
-        rg_r = rg_uvw(1)*cos(phi) + rg_uvw(2)*sin(phi)
-        r_gyro = r_gyro*(1 - rg_r/(2*R))
+        rg_r = rg_uvw(1) * cos(phi) + rg_uvw(2) * sin(phi)
```

Comment:
General style rule: Don't put spaces between `*` and `/` only for `+` and `-`. It looks cleaner that way